packages/yoga/src/Card/native/PlanCard/Actions.jsx

```diff
@@ -1,12 +1,40 @@
 import styled from 'styled-components';
 
 import Actions from '../Card/Actions';
+import Button from '../../../Button';
 import theme from '../../../Theme/helpers/themeReader';
 
-const Action = styled(Actions)`
-  margin-top: ${theme.components.card.plan.actions.margin.top}px;
+const {
+  components: {
+    card: {
+      plan: { actions },
+    },
+  },
+} = theme;
+
+const PlanCardButton = styled(Button).attrs({
+  small: true,
+  full: true,
+})``;
+
+const PlanCardButtonText = styled(Button.Text).attrs({
+  small: true,
+  full: true,
+})`
```

Comment:
Do we have any criteria on when do we use `attrs` and when do we just simply send these props directly in `<PlanCardButton small full />`?

(It's not change request, really just a doubt)

Reply:
I put `attrs` in this case because the `PlanCard.Button` and `PlanCard.ButtonText` always be small and full, then the user of the component will not need to add this `attrs` in `<PlanCard.Button small full>` every time she/he uses it. 

Do you think it is better to leave it to the user to add this manually?